exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -84,14 +85,15 @@ func createDefaultConfig() component.Config {
 	clientConfig.Timeout = exporterhelper.NewDefaultTimeoutConfig().Timeout
 
 	return &Config{
-		Namespace:         "",
-		ExternalLabels:    map[string]string{},
-		MaxBatchSizeBytes: 3000000,
-		TimeoutSettings:   exporterhelper.NewDefaultTimeoutConfig(),
-		BackOffConfig:     retrySettings,
-		AddMetricSuffixes: true,
-		SendMetadata:      false,
-		ClientConfig:      clientConfig,
+		Namespace:           "",
+		ExternalLabels:      map[string]string{},
+		MaxBatchSizeBytes:   3000000,
+		TimeoutSettings:     exporterhelper.NewDefaultTimeoutConfig(),
+		BackOffConfig:       retrySettings,
+		AddMetricSuffixes:   true,
+		SendMetadata:        false,
+		RemoteWriteProtoMsg: prometheusremotewrite.RemoteWriteProtoMsgV1,
```

Comment:
We should validate this field to make sure it is set to one of the enum options, if it is set.

Reply:
Done.